app/channels/application_cable/channel.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Let's make an exception to this magic comment in the rubocop file

Reply:
done!